.github/ISSUE_TEMPLATE/support.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Support Request
+about: Get some help using this repository
+labels: support
```

Comment:
I presume we have to create this label for this to work?

Reply:
Yes, but can be created after merge :)